release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -82,6 +82,10 @@ In {product-title} {product-version}, you can expand an installer provisioned cl
 [id="ocp-4-9-web-console"]
 === Web console
 
+[id="ocp-4-9-assessing-node-logs-from-the-node-detail-page"]
+==== Accessing node logs from the node detail page
```

Comment:
If the page's name is "Node Details" then it needs to be like this: **Node Details** page. This comment also applies to below. 

This is from our Documentation Guidelines:

> Use single asterisks for web console / GUI items (menus, buttons, page titles, etc.). Use two characters to form the arrow in a series of menu items (->).